Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp

```diff
@@ -417,11 +417,9 @@ static void webkitWebContextConstructed(GObject* object)
     API::ProcessPoolConfiguration configuration;
     configuration.setInjectedBundlePath(FileSystem::stringFromFileSystemRepresentation(bundleFilename.get()));
     configuration.setUsesWebProcessCache(true);
-#if PLATFORM(GTK)
+#if PLATFORM(GTK) && !USE(GTK4)
     configuration.setProcessSwapsOnNavigation(priv->psonEnabled);
-#if !USE(GTK4)
     configuration.setUseSystemAppearanceForScrollbars(priv->useSystemAppearanceForScrollbars);
-#endif
```

Comment:
Don't you need to set it to true for GTK4?

Reply:
Nope, that's the default set in WebPreferencesExperimental.yaml. I got confused by this too yesterday and had to check to be sure PSON was really enabled for WPE.

Reply:
Even for non developer builds? I still think it's safer to explicitly set it to true here.

Reply:
> Even for non developer builds?

I think so? I don't think the runtime features are hooked up to `ENABLE(DEVELOPER_MODE)`.

But we can explicitly set it, sure. It should hopefully not change anything. If it does, then PSON will be enabled for WPE for the first time.

Reply:
OK, done. Better?

Reply:
> I think so? I don't think the runtime features are hooked up to ENABLE(DEVELOPER_MODE).

I thought preferences in WebPreferencesExperimental were only enabled when `ENABLE(EXPERIMENTAL_FEATURES)`, but they aren't. In any case, it doesn't hurt to make it explicit, it's clearer and won't break if the preferences change in the future.